src/module/types/template/Matrix.ts

```diff
@@ -1,6 +1,7 @@
 import { ModifiableField } from "../fields/ModifiableField";
 import { AttributeField } from "./Attributes";
 import { ConditionData } from "./Condition";
+import { MatrixRules } from '@/module/rules/MatrixRules';
```

Comment:
this makes me a bit uneasy with circular imports... Just make sure nothing weird is happening... :D